### PR TITLE
added FAQ about "No space left on device" error

### PR DESCRIPTION
**Pull request by EwDa291**
Due to some problems with the local build, I unfortunately wasn't fully able to test the embedded links in the newly written section.

### CODE REVIEW

**Comment by boegel** [comment]
People will also run into this problem outside of jobs, so maybe we should park this under a new section, or rename "`Troubleshooting jobs`" to "`Troubleshooting`"?

File: mkdocs/docs/HPC/FAQ.md [hunk 1]
```diff
@@ -226,6 +226,24 @@ information, see .
...
+
+When trying to create files, errors like this can occur:
+
+```shell
+touch: cannot touch 'filename': No space left on device
+```
+
+The error "No space left on device" can mean two different things: 
+either all allocated memory on the file system in question has been used, 
+or the inode limit has been reached on that file system. An inode can be seen as a "file slot",
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:239]:
```suggestion
or the *inode limit* has been reached on that file system. An inode can be seen as a "file slot",

File: mkdocs/docs/HPC/FAQ.md [hunk 2]
```diff
@@ -226,6 +226,24 @@ information, see .
...
+### Why do I get a "No space left on device" error, while there is still available space?
+
+When trying to create files, errors like this can occur:
+
+```shell
+touch: cannot touch 'filename': No space left on device
+```
+
+The error "No space left on device" can mean two different things: 
+either all allocated memory on the file system in question has been used,
```
[the remark this answers is not in the document]
Comment (boegel) [reply mkdocs/docs/HPC/FAQ.md:238]: I would also make this two into an actual list, which will make them stand out a bit better.

File: mkdocs/docs/HPC/FAQ.md [hunk 3]
```diff
@@ -226,6 +226,24 @@ information, see .
...
+When trying to create files, errors like this can occur:
+
+```shell
+touch: cannot touch 'filename': No space left on device
+```
+
+The error "No space left on device" can mean two different things: 
+either all allocated memory on the file system in question has been used, 
+or the inode limit has been reached on that file system. An inode can be seen as a "file slot", 
+meaning that when the limit is reached, no more files can be created. There is a standard inode limit in place that will be expanded if needed.
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:240]:
```suggestion
meaning that when the limit is reached, no more additional files can be created.
There is a standard inode limit in place that will be increased if needed.

File: mkdocs/docs/HPC/FAQ.md [hunk 4]
```diff
@@ -226,6 +226,24 @@ information, see .
...
+
+```shell
+touch: cannot touch 'filename': No space left on device
+```
+
+The error "No space left on device" can mean two different things: 
+either all allocated memory on the file system in question has been used, 
+or the inode limit has been reached on that file system. An inode can be seen as a "file slot", 
+meaning that when the limit is reached, no more files can be created. There is a standard inode limit in place that will be expanded if needed. 
+The number of inodes used per file system can be checked on <https://account.vscentrum.be>.
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:241]:
```suggestion
The number of inodes used per file system can be checked on [the VSC account page](https://account.vscentrum.be).

File: mkdocs/docs/HPC/FAQ.md [hunk 5]
```diff
@@ -226,6 +226,24 @@ information, see .
...
+
+The error "No space left on device" can mean two different things: 
+either all allocated memory on the file system in question has been used, 
+or the inode limit has been reached on that file system. An inode can be seen as a "file slot", 
+meaning that when the limit is reached, no more files can be created. There is a standard inode limit in place that will be expanded if needed. 
+The number of inodes used per file system can be checked on <https://account.vscentrum.be>. 
+
+Possible solutions to this problem include cleaning up unused files and directories or 
+[compressing directories with a lot of files into zip- or tar-files](../linux-tutorial/manipulating_files_and_directories/?h=zip#zipping-gzipgunzip-zipunzip). 
+If the problem persists, feel free to contact support [here](./#i-have-another-questionproblem).
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:245]:
```suggestion
If the problem persists, feel free to [contact support](./#i-have-another-questionproblem).

File: mkdocs/docs/HPC/FAQ.md [hunk 6]
```diff
@@ -226,6 +226,26 @@ information, see .
 
 {% endif %}
 
+
+### Why do I get a "No space left on device" error, while I still have storage space left?
+
+When trying to create files, errors like this can occur:
+
+```shell
+touch: cannot touch 'filename': No space left on device
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:235]:
```suggestion
touch: cannot touch 'filename': No space left on device

File: mkdocs/docs/HPC/FAQ.md [hunk 7]
```diff
@@ -226,6 +226,26 @@ information, see .
...
+```
+
+The error "`No space left on device`" can mean two different things:
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:238]: You'll need to add an empty line to make sure list is rendered correctly (also after last bullet)

```suggestion
The error "`No space left on device`" can mean two different things:

```

File: mkdocs/docs/HPC/FAQ.md [hunk 8]
```diff
@@ -226,6 +226,26 @@ information, see .
...
+- all available *storage quota* on the file system in question has been used;
+- the *inode limit* has been reached on that file system.
+An *inode* can be seen as a "file slot", meaning that when the limit is reached, no more additional files can be created.
+There is a standard inode limit in place that will be increased if needed. 
+The number of inodes used per file system can be checked on [the VSC account page](https://account.vscentrum.be).
+
+Possible solutions to this problem include cleaning up unused files and directories or 
+[compressing directories with a lot of files into zip- or tar-files](../linux-tutorial/manipulating_files_and_directories/?h=zip#zipping-gzipgunzip-zipunzip). 
+If the problem persists, feel free to [contact support](./#i-have-another-questionproblem).
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:247]:
```suggestion

If the problem persists, feel free to [contact support](./#i-have-another-questionproblem).

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by EwDa291** [comment]
@boegel should be ready to merge now

**Review by boegel — APPROVED** [review]

**Pull request merged**